apollo-ios/Sources/ApolloAPI/FragmentProtocols.swift

```diff
@@ -9,6 +9,14 @@ public protocol Fragment: SelectionSet, Deferrable {
   static var fragmentDefinition: StaticString { get }
 }
 
+/// Extension providing default implementation for the ``Fragment`` protocol.
+extension Fragment {
+  // Default implementation for the `fragmentDefinition` variable
+  static var fragmentDefinition: StaticString {
```

Comment:
Does this need to be `public`?

Reply:
I think the failing CI test is because of this actually.

Reply:
It does indeed, forgot to mark it public